013_odd_Or_Even_program.js

```diff
@@ -0,0 +1,11 @@
+// program to check whether the entered number is even or  odd
```

Comment:
Please write it as a function. you should declare  the function and call it from your program